### PR TITLE
77 user   profile page

### PR DESCRIPTION
**Pull request by R-Thibault**

### CODE REVIEW

File: frontend/src/pages/profile.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
+import Layout from "@/components/organisms/layout";
+import React, { FormEvent, useEffect, useState } from "react";
+import Image from "next/image";
+import { useMutation, useQuery } from "@apollo/client";
+import { MY_PROFILE, UPDATE_USER } from "@/Request/user";
+import { UserType, UserUpdateType } from "@/types/user.type";
+import { spawn } from "child_process";
+
+export default function profile() {
+  const [error, setError] = useState<"emptyFields">();
```
Comment (MathieuHERY) [inline frontend/src/pages/profile.tsx]: Modifier le typage des useStates 
const [error, setError] = useState<Boolean>(false)

File: frontend/src/pages/profile.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  const [success, setSuccess] = useState<"succes">();
```
Comment (MathieuHERY) [inline frontend/src/pages/profile.tsx]: idem ici => Modifier le typage des useStates
const [success, setSuccess] = useState<Boolean>(false)

**Review by MathieuHERY — CHANGES_REQUESTED** [review]

File: frontend/src/pages/profile.tsx (hunk 1 above)
Comment (R-Thibault) [reply frontend/src/pages/profile.tsx]: Done

File: frontend/src/pages/profile.tsx (hunk 2 above)
Comment (R-Thibault) [reply frontend/src/pages/profile.tsx]: Done

**Review by MathieuHERY — APPROVED** [review]

**Pull request merged**